Watch Extension/FeedingView.swift

```diff
@@ -92,7 +55,7 @@ struct FeedingView: View {
             ActionSheet(title: Text("What do you want to do?"),
                         buttons: [
                             .default(Text("Stop")) {
-                                self.communicate(type: self.feeding.type, side: self.feeding.side, action: .stop)
+                                self.store.send(.communicate(type: self.feeding.type, side: self.feeding.side, action: .stop))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/d0969e50f3967ea66270aaf4bf0f82ba44576d07/Watch Extension/FeedingView.swift#L58"></span>Line should be 120 characters or less: currently 126 characters<br />
<code>line_length</code> <code>FeedingView.swift:58</code></td>
    </tr>
  </tbody>
</table>